portfolio/src/main/java/com/google/sps/comments/Comment.java

```diff
@@ -41,4 +45,15 @@ public String getMessage() {
     return message;
   }
 
+  public long getId() {
+    return id;
+  }
+
+  public long getVotes(){
+    return votes; 
+  }
+
+  public void setVotes(long vote){
```

Comment:
I'd use another name for this method.
"setVotes" sounds to me as overwriting the current value of "votes", but it seems that it is doing an addition. Maybe change it to "addVotes".